apps/change-log.md

```diff
@@ -14,6 +14,16 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2022-12-01
+
+### Changes with software release 7.78.0
```

Comment:
https://epages.atlassian.net/browse/UNITY-7991
Was it released with 7.77.0?

Reply:
@bolsu Good point. I think you are right. I will correct the PR accordingly.